Formula/parallel.rb

```diff
@@ -1,17 +1,17 @@
 class Parallel < Formula
   desc "Shell command parallelization utility"
   homepage "https://savannah.gnu.org/projects/parallel/"
-  url "https://ftp.gnu.org/gnu/parallel/parallel-20200822.tar.bz2"
-  mirror "https://ftpmirror.gnu.org/parallel/parallel-20200822.tar.bz2"
-  sha256 "9654226a808392c365b1e7b8dea91bf4870bc4f306228d853eb700679e21be09"
-  license "GPL-3.0-or-later"
+  url "https://ftp.gnu.org/gnu/parallel/parallel-20200722.tar.bz2"
+  mirror "https://ftpmirror.gnu.org/parallel/parallel-20200722.tar.bz2"
+  sha256 "4801d44f2f71eed26386a0623a6fb3cadd7fa7ec2b5a7bbc5b7b52e2a0450d6f"
+  license "GPL-3.0"
```

Comment:
Do not revert changes in license

```suggestion
  license "GPL-3.0-or-later"
```